Feat/update member capstone group

minhdev313 opened the pull request:
Nhờ Minh review nhé

minhdev313 committed:
Create update member capstone group ( include add and remove group)

minhdev313 committed:
add swagger

minhdev313 committed:
validate leader remove case

minhbear commented on internal/controller/capstone_group/capstone_group.controller.go:
```diff
@@ -323,3 +324,49 @@ func (cgc *CapstoneGroupController) GetListStudentHaveCapstoneGroup(ctx *gin.Con
 	}
 	response.SuccessResponse(ctx, http.StatusOK, output)
 }
+
```
Please add this controller below the same path to easy review or check duplicate router

minhbear commented on internal/controller/capstone_group/capstone_group.controller.go:
```diff
@@ -323,3 +324,49 @@ func (cgc *CapstoneGroupController) GetListStudentHaveCapstoneGroup(ctx *gin.Con
+// @Security ApiKeyAuth
+func (ecc *CapstoneGroupController) UpdateCapstoneGroupStudent(ctx *gin.Context) {
+	idParam := ctx.Param("capstone_group_id")
+	capstoneGroupID, err := strconv.ParseInt(idParam, 10, 64)
```
use this capstoneGroupID, err := strconv.Atoi(idParam) same with other

minhdev313 committed:
Update func UpdateCapstone GroupStudent

-Update xử lý add nhiều học sinh cùng 1 lúc
-Remove xử lý remove leader và update leader
-Fix 2 comment ở capstone_group.controller
